### PR TITLE
doc(i-am-tc) : Custom model en translation

### PR DESCRIPTION
**Pull request by i-am-tc**
Lines with potentially problematic translations or requiring some vetting.

38
48
53 (为一维的情况)
160
279
303

### CODE REVIEW

File: source/04_best_practice/custom_model_en.rst [hunk 1]
```diff
@@ -0,0 +1,343 @@
+如何自定义神经网络模型（model）
```
Comment (PaParaZz1) [inline source/04_best_practice/custom_model_en.rst:1]: For English version, it is ok to just be named as `custom_model.rst`, and you should add this file in corresponding `index.rst`

File: source/04_best_practice/custom_model_en.rst [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+
+Observe here that the method either returns \ ``'maqac_continuous', ['ding.model.template.maqac']``\ or \ ``'qac', ['ding.model.template.qac']``\. In both cases, the first item in the return tuple is the name registered with DI-engine's model registry mechanism. The second item gives an indication of the file path of where the model file is located.
+
+在使用配置文件时，DI-engine 封装好的入口文件将会把 ``cfg.policy.model`` 中的参数，逐个传给默认注册好的模型类（例如把 ``obs_shape``, ``action_shape`` 等参数传给 
+`QAC <https://github.com/opendilab/DI-engine/blob/main/ding/model/template/qac.py#L13>`_ ），模型类中根据传入参数自动生成所需的神经网络（例如为向量输入使用全连接层（FC）而为图像输入使用卷积（Conv））。
+
+When using the configuration file ``cfg.policy.model``, DI-engine will sequentially pass each parameter into the model registered with DI-engine's registry mechanism. (For example, parameters ``obs_shape``, ``action_shape`` etc will be passed into `QAC <https://github.com/opendilab/DI-engine/blob/main/ding/model/template/qac.py#L13>`_ ). The required neural network is then automatically generated in the model class based on the incoming parameters (e.g. a fully connected layer (FC) for vector input and a convolution (Conv) for image input).
```
Comment (PaParaZz1) [inline source/04_best_practice/custom_model_en.rst:41]: parameter -> argument
sequentially -> correspondingly

File: source/04_best_practice/custom_model_en.rst [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+
+而此时查阅 \ `sac 源码 <https://github.com/opendilab/DI-engine/blob/main/ding/policy/sac.py>`__\ 可知 \ ``default_model``\ 为 \ `qac <https://github.com/opendilab/DI-engine/blob/main/ding/model/template/qac.py>`__\ ，
+\ ``qac model``\ 中暂时只支持 \ ``obs_shape``\ 为一维的情况，此时我们即可根据需求自定义 model 并应用到 policy。
+
+If one were to look at the source code of \ `sac <https://github.com/opendilab/DI-engine/blob/main/ding/policy/sac.py>`__\, it can be seen that the \ ``default_model``\ is actually \ `qac <https://github.com/opendilab/DI-engine/blob/main/ding/model/template/qac.py>`__\. The \ ``qac model``\ currently only supports an \ ``obs_shape``\ of 1 only. Hence, it becomes apparent that one must customize a model according to one's needs and ensure that the policy is setup accordingly.
```
Comment (PaParaZz1) [inline source/04_best_practice/custom_model_en.rst:56]: The \ ``qac model``\ currently only supports an \ ``obs_shape``\ of one dimension (e.g. `(4, )`)

File: source/04_best_practice/custom_model_en.rst [hunk 4]
```diff
@@ -0,0 +1,343 @@
...
++-----------------------+-------------------------------------+
+|MultiHead              |处理动作空间为多维的情况             |
++-----------------------+-------------------------------------+
+Output discrete action value
+Output Q value distribution
+Output Q value distribution
+Quantile Regression DQN, output action quantiles
+Output action quantiles
+Output discrete action value logits
+Output continuous action Q values
```
Comment (PaParaZz1) [inline source/04_best_practice/custom_model_en.rst:172]: There is a bit confusing in original Chinese version.
It should be: 用于输出价值函数值（Value）或回归预测连续动作值

File: source/04_best_practice/custom_model_en.rst [hunk 5]
```diff
@@ -0,0 +1,343 @@
...
+
+-  旧pipeline
+
+-  Old pipeline
+
+将定义好的 model 作为参数传入 \ `serial_pipeline <https://github.com/opendilab/DI-engine/blob/main/ding/entry/serial_entry.py#L22>`__\ , 
+传入的 model 将在 \ `serial_pipeline <https://github.com/opendilab/DI-engine/blob/main/ding/entry/serial_entry.py#L59>`__\ 
+通过 \ ``create_policy``\  被调用。或者跟上述新 pipeline 一样，作为参数传入 policy 。
+
+Pass the defined model into \ `serial_pipeline <https://github.com/opendilab/DI-engine/blob/main/ding/entry/serial_entry.py#L22>`__\ as a argument. The model will then be passed on to \ ``create_policy``\.
```
Comment (PaParaZz1) [inline source/04_best_practice/custom_model_en.rst:281]: There is a redundant sentence "或者跟上述新 pipeline 一样，作为参数传入 policy", you can ignore and remove it

**Review by PaParaZz1 — CHANGES_REQUESTED** [review]

File: source/04_best_practice/custom_model_en.rst (hunk 1 above)
Comment (i-am-tc) [reply source/04_best_practice/custom_model_en.rst]: Done with [5204a36](https://github.com/opendilab/DI-engine-docs/pull/194/commits/5204a3607a0d3dbba11d1e933651c27165802973) and [3063e43](https://github.com/opendilab/DI-engine-docs/pull/194/commits/3063e430e247650fa13b15febe147128e35c865c)

File: source/04_best_practice/custom_model_en.rst (hunk 2 above)
Comment (i-am-tc) [reply source/04_best_practice/custom_model_en.rst]: Done with https://github.com/i-am-tc/DI-engine-docs/commit/1211618de5375b24008ded59427d6dcdebe4afe8

File: source/04_best_practice/custom_model_en.rst (hunk 3 above)
Comment (i-am-tc) [reply source/04_best_practice/custom_model_en.rst]: Done with https://github.com/i-am-tc/DI-engine-docs/commit/cf37040ba47a6b46809c1941f77245aa1768b916

File: source/04_best_practice/custom_model_en.rst (hunk 4 above)
Comment (i-am-tc) [reply source/04_best_practice/custom_model_en.rst]: I did a bit of search and it seems "regression" and "quantile regression" are dfferent, which means "回归" and "分位数回归" , being directly translation, should also have different meanings of their own.

I prefer " ... 动作值". If I don't remember wrongly, the Q in DQN refers to action value, not state value. I remember to be different, though they can be related together.

Hence, I took the liberty and made the following edit decision
* I changed the Chinese to : 分位数回归预测连续动作值
* I changed the English to : Quantile regression continuous action value

PS: I was deciding between "Quantile regression" vs "Quantile regressed" but decided to keep to continuous tense.

File: source/04_best_practice/custom_model_en.rst (hunk 5 above)
Comment (i-am-tc) [reply source/04_best_practice/custom_model_en.rst]: Done with https://github.com/i-am-tc/DI-engine-docs/commit/bce52c7e0ceee64ae91015d818b0db333112a67b

**Review by PaParaZz1 — APPROVED** [review]

**Pull request merged**